bindings/libdnf5/conf.i

```diff
@@ -128,7 +128,10 @@ wrap_unique_ptr(StringUniquePtr, std::string);
 import re
 
 def _config_option_getter(config_object, option_name):
-    return getattr(config_object, option_name)().get_value()
+    try:
+        return getattr(config_object, option_name)().get_value()
+    except RuntimeError:
```

Comment:
I think it is not ideal. The except part quite wide, but this is a consequence of unfinished error handling in swig part. What about to add a todo to use more targeted error handling when it will be available?

Reply:
I know it's broad, but do we actually want to throw any exception when accessing an attribute like this in Python? Is there some case we want to throw an exception?

Reply:
I am not sure whether it is valid, but I see ways how to generate an exception for different case that option is not set. I don't know what happen when attribute does not exists. If it generates runtime error then it is expected that it will fail.

Reply:
Thank you for clarification.